Sources/SpeziLLMLocal/LLMLocalSession+Generate.swift

```diff
@@ -96,16 +103,12 @@ extension LLMLocalSession {
         )
         
         await MainActor.run {
-            if schema.injectIntoContext {
-                // Yielding every Nth token may result in missing the final tokens.
-                let reaminingTokens = result.tokens.count % schema.parameters.displayEveryNTokens
-                let lastTokens = Array(result.tokens.suffix(reaminingTokens))
-                let text = tokenizer.decode(tokens: lastTokens)
-                continuation.yield(text)
-            }
-            
             context.append(assistantOutput: result.output, complete: true)
```

Comment:
Shouldn't this only happen in case of `schema.injectIntoContext` = true? See comment above